examples/PyOpenCL/build.sh

```diff
@@ -4,4 +4,4 @@ TS_BASEDIR="$1"
 TS_BUILDDIR="$2"
 TS_SRCDIR="$3"
 
-source "${TS_BUILDDIR}/bin/activate" && cd "${TS_SRCDIR}" && python3 setup.py build
```

Comment:
From https://docs.python.org/3/library/venv.html

> You don’t specifically need to activate an environment; activation just prepends the virtual environment’s binary directory to your path, so that “python” invokes the virtual environment’s Python interpreter and you can run installed scripts without having to use their full path. However, **all scripts installed in a virtual environment should be runnable without activating it, and run with the virtual environment’s Python automatically.**

1) slightly shorter code 2) explicit paths instead of env variables

Reply:
ok